style.css

```diff
@@ -418,16 +446,16 @@ strong {
   font-size: 0.1rem;
 }
 
-.contact {
+.contactSection {
   display: flex;
   flex-direction: column;
   width: 100%;
   background-color: #6070ff;
   border-top-left-radius: 8rem;
 }
 
-.contact-bg-img {
```

Comment:
- [ ] For the contact section as well as mentioned by the previous reviewer please use the normal font-weight instead of bold

<img width="585" alt="Screenshot 2023-07-15 at 7 09 24 PM" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/44798044/65189ea7-e943-4489-9e48-b9cf89e30c63">